docs/guides/install-python.md

```diff
@@ -120,6 +120,27 @@ To force uv to use the system Python, provide the `--no-managed-python` flag. Se
 [Python version preference](../concepts/python-versions.md#requiring-or-disabling-managed-python-versions)
 documentation for more details.
 
+## Upgrading Python patch versions
```

Comment:
We shouldn't repeat the concept documentation verbatim. The guide should be simple and the concept should have more details.

Reply:
Updated